src/pylorax/api/compose.py

```diff
@@ -404,8 +404,22 @@ def start_build(cfg, yumlock, gitlock, branch, recipe_name, compose_type, test_m
         raise RuntimeError("Problem depsolving %s: %s" % (recipe["name"], str(e)))
     log.debug("installed_size = %d, template_size=%d", installed_size, template_size)
 
+    with yumlock.lock:
+        repos = yumlock.yb.repos.listEnabled()
+    if not repos:
+        raise RuntimeError("No enabled repos, canceling build.")
+
+    # Anaconda also stores the metadata on the disk once it is partitioned, try to take this into account
+    metadata_size = 0
```

Comment:
Is this accurate enough? I think I trust examining the filesystem more than yum tracking things.

Reply:
Well, I only counted these three metadata files, because I download these ones when I do a "yum update". But I agree with you, it can be as accurate as the metadata downloaded in "installroot".